class_person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :age, :name, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
The id has been added @shahadat3669 thanks. However, I couldn't see any reference in the possibility to use "this" method to set the id attribute when creating a new Person object.

Reply:
- [ ] 👍 The `can_use_services` method is currently marked as private, which means it can't be accessed from outside the class. To make it accessible externally, please consider changing its access modifier to public. This way, other parts of your code can utilize this method as intended. Keep up the great work! 🚀

Check the example bellow:
```
person = Person.new(10, 22, name='maximilianus')
person.can_use_services

```

![image](https://github.com/diarisdiakite/oop_school-library/assets/55840999/9749c3fe-32af-4057-b557-268c360946d5)